robottelo/cli/base.py

```diff
@@ -1,13 +1,14 @@
 # -*- encoding: utf-8 -*-
-"""
-Generic base class for cli hammer commands
-"""
-
+"""Generic base class for cli hammer commands."""
 import logging
+import re
 
 from robottelo.cli import hammer
 from robottelo.common import conf, ssh
 
+TASK_STATUS_REGEX = re.compile(
```

Comment:
Mind adding a comment with an example of what the output is? This way it will be easier to maintain this regex in the future.